emscripten.py

```diff
@@ -31,6 +32,9 @@
 from tools.shared import treat_as_user_export
 from tools.settings import settings
 
+sys.path.append(path_from_root('third_party'))
```

Comment:
Is this line needed?  I would expect all the `tools` imports above to fail if it was

Reply:
Why is this related to all the `tools` lines above?

This can be removed as you suggested, because `from tools import building` above eventually add `third_party` directory to `sys.path`:
https://github.com/emscripten-core/emscripten/blob/3cd9a380124bf17845f46819a895ec55a9e6ecf8/emscripten.py#L22
https://github.com/emscripten-core/emscripten/blob/3cd9a380124bf17845f46819a895ec55a9e6ecf8/tools/building.py#L23
https://github.com/emscripten-core/emscripten/blob/3cd9a380124bf17845f46819a895ec55a9e6ecf8/tools/webassembly.py#L18

But it is not easy to see this by just looking at `emscripten.py`. Not sure if we should depend on that `from tools import building` line is above?